### PR TITLE
Add REA to default config

### PR DESCRIPTION
**Pull request by lizmat**
This should completely activate BackPan for Raku in zef

### CODE REVIEW

**Comment by ugexe** [comment]
Need to `git grep "zef --debug --/fez --/cpan --/p6c install Base64"` and add a `--/rea` flag to each instance

**Comment by lizmat** [comment]
Wow, hadn't realized that AppVeyor actually compiles MoarVM from scratch?

**Comment by lizmat** [comment]
Whee!  Green!  :-)

**Comment by ugexe** [comment]
One last thing that is needed is a test. All the ecosystems use the same template -- see: https://github.com/ugexe/zef/blob/7d6b48db23eaca06c332fa6d0ca8ecae80ae83d4/xt/repository.t#L46-L66

File: xt/repository.t [hunk 1]
```diff
@@ -110,4 +110,26 @@ subtest  'Ecosystems => fez' => {
...
+    my @backends = [
+        { module => "Zef::Service::Shell::git" },
+        { module => "Zef::Service::Shell::wget" },
+        { module => "Zef::Service::Shell::curl" },
+        { module => "Zef::Service::Shell::PowerShell::download" },
+    ];
+
+    my $fetcher = Zef::Fetch.new(:@backends);
+    my $cache   = $*HOME.child('.zef/store') andthen { mkdir $_ unless $_.IO.e };
+    my $p6c     = Zef::Repository::Ecosystems.new(name => 'rea', :$fetcher, :$cache, :auto-update, :@mirrors);
```
Comment (ugexe) [inline xt/repository.t:125]: Please rename the `$p6c` to `$rea`

**Comment by lizmat** [comment]
Done as suggested.

**Pull request merged**